docs/src/reference/gremlin-applications.asciidoc

```diff
@@ -2336,6 +2336,44 @@ List<Vertex> results = g.V().hasLabel("person").elementMap('name').toList();
 
 Both of the above requests return a list of `Map` instances that contain the `id`, `label` and the "name" property.
 
+*Compatibility*
+
+*It is not recommended to use older driver versions with 3.7.x Gremlin Server*, as some older drivers do not construct
```

Comment:
```suggestion
*It is not recommended to use 3.6.x or below driver versions with 3.7.x or above Gremlin Server*, as some older drivers do not construct
```
This might be a bit more future proof as new releases go out.